Makefile

```diff
@@ -325,12 +325,14 @@ check_critic:
 #-------------#
 
 build_taxonomies:
-	@echo "🥫 build taxonomies on ${CPU_COUNT} procs"
-	${DOCKER_COMPOSE} run --no-deps --rm backend make -C taxonomies -j ${CPU_COUNT}
+	@echo "🥫 build taxonomies"
+	${DOCKER_COMPOSE} run --no-deps --rm -e GITHUB_TOKEN=${GITHUB_TOKEN} backend /opt/product-opener/scripts/build_tags_taxonomy.pl ${name}
 
-rebuild_taxonomies:
-	@echo "🥫 re-build all taxonomies on ${CPU_COUNT} procs"
-	${DOCKER_COMPOSE} run --rm backend make -C taxonomies all_taxonomies -j ${CPU_COUNT}
+rebuild_taxonomies: build_taxonomies
+
+build_taxonomies_test:
+	@echo "🥫 build taxonomies"
+	${DOCKER_COMPOSE_TEST} run --no-deps --rm -e GITHUB_TOKEN=${GITHUB_TOKEN} backend /opt/product-opener/scripts/build_tags_taxonomy.pl ${name}
```

Comment:
I don't think we need a specific target for tests because we bind mount the taxonomies folder (so it's shared between tests and dev)